scrapers/MissaX.yml

```diff
@@ -9,22 +9,22 @@ xPathScrapers:
     scene:
       Title: //p[@class="raiting-section__title"]/text()
       Date:
-        selector: //p[@class="dvd-scenes__data"][1]/text()[1]
+        selector: //p[@class="dvd-scenes__data" and contains(text(), " Added:")]
         postProcess:
           - replace:
               - regex: (?:.+Added:\s)([\d/]*).+
                 with: $1
           - parseDate: 01/02/2006
-      Details: //strong/text()
+      Details: //p[@class='text text--marg']//strong
```

Comment:
The Details part needs a little more testing.
From a couple of samples i had `https://missax.com/tour/trailers/A-Killer-On-The-Loose-pt-3.html`,`https://missax.com/tour/trailers/The-Family-Secret.html`
```yaml
      Details:
        selector: //p/strong
        concat: "\n"
```
Seems to work better

Reply:
Ah yes I did not see the extra paragraphs in the ones I looked at. 
Updated :-)